router/gin/engine.go

```diff
@@ -73,7 +73,6 @@ func NewEngine(cfg config.ServiceConfig, opt EngineOptions) *gin.Engine {
 				engine.AppEngine = ginOptions.AppEngine
 				engine.MaxMultipartMemory = ginOptions.MaxMultipartMemory
 				engine.RemoveExtraSlash = ginOptions.RemoveExtraSlash
-				engine.UseH2C = ginOptions.UseH2C
```

Comment:
Since Lura has already "published" / "made public" the `use_h2c` flag just for the gin router, we need to keep it: take into account that even `krakend-ce` is the main project using the Lura framework, we cannot be sure that any other software is using the library and expecting this flag to keep working. So is better that we keep this.